index.html

```diff
@@ -20,6 +20,8 @@ <h2>Tracker Reporting</h2>
 	<li><a href="./tracker-reporting/1major-via-img.html">1 major tracker loaded via img</a></li>
 	<li><a href="./tracker-reporting/document-fragment.html">Image loaded via document fragment</a> <a href="./tracker-reporting/document-fragment.html?delay=5000">[Delay 5s]</a></li>
 	<li><a href="./tracker-reporting/1major-via-fetch.html">1 major tracker loaded via fetch</a> <a href="./tracker-reporting/1major-via-fetch.html?delay=5000">[Delay 5s]</a></li>
+	<li><a href="./tracker-reporting/option-blocked-tracker.html">Tracker blocked by options on bad.third-party.site</a> <a href="http://bad.third-party.site/tracker-reporting/option-blocked-tracker.html"></a></li>
+	<li><a href="./tracker-reporting/option-blocked-tracker.html">Tracker not blocked by options on good.third-party.site</a> <a href="http://good.third-party.site/tracker-reporting/option-blocked-tracker.html"></a></li>
```

Comment:
I'd make it a one page and link from index.html to two domains
"Blocklist site-specific blocking ('options blocking')"